demos/custom-tabs-session/build.gradle

```
@@ -0,0 +1,50 @@
+/*
+ *    Copyright 2020 Google LLC
```

```suggestion
 *    Copyright 2021 Google LLC
```

2020 is finally over :P